app/component/itinerary/ItineraryPage.js

```diff
@@ -838,6 +850,20 @@ export default function ItineraryPage(props, context) {
     altStates[PLANTYPE.BIKETRANSIT][0].plan,
   ]);
 
+  // merge direct car and car transit plans into one
+  useEffect(() => {
+    if (
+      altStates[PLANTYPE.CAR][0].loading === LOADSTATE.DONE &&
+      altStates[PLANTYPE.CARTRANSIT][0].loading === LOADSTATE.DONE
+    ) {
+      const plan = mergeCarDirectAndTransitPlans(
```

Comment:
I guess this is not OK. car + transit should be set only when there are carBoardingModes configured and settings.includeCarSuggestions is true.

Reply:
This is very similar to how the mergeBikeTransitPlans works. Because the check for making the cartransit query is made in planParamUtil, this should be fine as is. Do you want me to change this and the way it works for bikes or should I leave both as is?

Reply:
I now changed it to
altStates[PLANTYPE.CAR][0].loading === LOADSTATE.DONE &&
altStates[PLANTYPE.CARTRANSIT][0].loading === LOADSTATE.DONE &&
settings.includeCarSuggestions &&
config.carBoardingModes !== undefined